src/Orleans.Runtime/Core/InsideRuntimeClient.cs

```diff
@@ -725,7 +725,13 @@ private Task OnRuntimeInitializeStart(CancellationToken tc)
             typeManager.Start();
             GrainTypeResolver = typeManager.GetTypeCodeMap();
             stopWatch.Stop();
-            this.logger.Info(ErrorCode.SiloStartPerfMeasure, $"Start InsideRuntimeClient took {stopWatch.ElapsedMilliseconds} Milliseconds");
+            var report = new LifeCyclePerfReport()
```

Comment:
What about storing the ```TimeSpan``` value in the report instead of the formatted string, since that string can be generated from the ```TaskName``` and the ```Duration``` (if we call it that).

Reply:
I think it is more flexible to store strings, it give user more flexibility to report metrics more than just time.

Reply:
But then we loose the most important part of perf, the time itself.